www/store/features/user/userSlice.js

```diff
@@ -127,11 +127,13 @@ export const userLogout = () => {
     const cookie = document.cookie
       .match( new RegExp( "(^| )" + "auth-token" + "=([^;]+)" ) );
 
+    console.log( cookie );
```

Comment:
```suggestion
```
We prolly don't wanna log this, i dont know if someone could grab a cookie from a log.